tutorial/developing_packages/editable_packages.rst

```diff
@@ -0,0 +1,204 @@
+
+    $ cd say
+
+    # Windows: we will build 2 configurations to show multi-config
+    $ conan install . -s build_type=Release
+    $ conan install . -s build_type=Debug
+    $ cmake --preset default
+    $ cmake --build --preset release
+    $ cmake --build --preset debug
+
+    # Linux, MacOS: we will only build 1 configuration
+    $ conan install .
+    $ cmake --preset release
+    $ cmake --build --preset release
+
+
+Using say/1.0 package in editable mode
+--------------------------------------
+
+Consuming a package in editable mode is transparent from the consumer perspective.
+In this case we can build the ``hello`` application as usual:
+
+.. code-block:: bash
+
+    $ cd ../hello
+
+    # Windows: we will build 2 configurations to show multi-config
+    $ conan install . -s build_type=Release
+    $ conan install . -s build_type=Debug
+    $ cmake --preset default
+    $ cmake --build --preset release
+    $ cmake --build --preset debug
+    $ build\Release\hello.exe
+    say/1.0: Hello World Release!
+    ...
+    $ build\Debug\hello.exe
+    say/1.0: Hello World Debug!
+    ...
+
+    # Linux, MacOS: we will only build 1 configuration
+    $ conan install .
+    $ cmake --preset release
+    $ cmake --build --preset release
+    $ ./build/Release/hello
+    say/1.0: Hello World Release!
+
+As you can see, ``hello`` can successfully find ``say/1.0`` header and library files.
+
+
+Working with editable packages
+------------------------------
+
+Once the above steps have been completed, you can work with your build system or IDE
+without involving Conan and make changes to the editable packages. The consumers will use
+those changes directly. Let's see how this works by making a change in the ``say`` source
+code:
+
+.. code-block:: bash
+
+    $ cd ../say
+    # Edit src/say.cpp and change the error message from "Hello" to "Bye"
+
+    # Windows: we will build 2 configurations to show multi-config
+    $ cmake --build --preset release
+    $ cmake --build --preset debug
+
+    # Linux, MacOS: we will only build 1 configuration
+    $ cmake --build --preset release
+
+
+And build and run the "hello" project:
+
+.. code-block:: bash
+
+    $ cd ../hello
+
+    # Windows
+    $ cd build
+    $ cmake --build --preset release
+    $ cmake --build --preset debug
+    $ Release\hello.exe
+    say/1.0: Bye World Release!
+    $ Debug\hello.exe
+    say/1.0: Bye World Debug!
+
+    # Linux, MacOS
+    $ cmake --build --preset release
+    $ ./hello
+    say/1.0: Bye World Release!
+
+
+In this manner, you can develop both the ``say`` library and the ``hello`` application
+simultaneously without executing any Conan command in between. If you have both open in
+your IDE, you can simply build one after the other.
+
+.. note::
+
+    When a package is in editable mode, some commands will not work. It is not
+    possible to :command:`conan upload`, :command:`conan export` or :command:`conan
+    create` when a package is in editable mode.
```

Comment:
Not true, it is possible now. Especially the ``conan create`` might work great, testing the package too